src/AutSoft.Core/AutSoft.Common.csproj

```diff
@@ -8,6 +8,8 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.Extensions.DependencyInjection.Abstractions" Version="6.0.0" />
+    <PackageReference Include="Microsoft.EntityFrameworkCore" Version="6.0.8" />
```

Comment:
Shouldn't we remove Version attributes here, as we have a Directory.Packages.props file?

Reply:
Fixed
From now all project in the src folder use the Directory.Packages.props file.